src/main/src/main/java/org/geoserver/config/impl/JAIEXTInfoImpl.java

```diff
@@ -0,0 +1,90 @@
+/* (c) 2015 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.config.impl;
+
+import it.geosolutions.jaiext.ConcurrentOperationRegistry.OperationItem;
+import it.geosolutions.jaiext.JAIExt;
+
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.geoserver.config.JAIEXTInfo;
+
+public class JAIEXTInfoImpl implements JAIEXTInfo {
+
+    /**
+     * Available JAI operations
+     */
+    public static final Set<String> JAI_OPS = new TreeSet<String>();
+
+    private Set<String> jaiOperations = JAI_OPS;
+    
+    /**
+     * Available JAIEXT operations
+     */
+    public static final TreeSet<String> JAIEXT_OPS = new TreeSet<String>();
+    
+    private Set<String> jaiExtOperations = JAIEXT_OPS;
+
+    static{
+        JAIExt.initJAIEXT();
+        List<OperationItem> jaiextOps = JAIExt.getJAIEXTOperations();
+        for(OperationItem item : jaiextOps){
+            String name = item.getName();
+            if (name.equalsIgnoreCase("algebric") || name.equalsIgnoreCase("operationConst")
```

Comment:
Hum... one more or and you don't need two branches? Or do we have a bug in the branches code? (It's the same)